### DISCUSSION TITLE
Help! Rewrite /flood to /flood/ in the URI

### DISCUSSION THREAD

**Discussion by jdarias** [jesec/flood#234] [Q&A]
I'm following the wiki in order to serve flood through nginx but I've stumbled upon the trailing / issue... wiki says:
> You may configure your web server to redirect users from /flood to /flood/. It is not covered by this tutorial.

Why not?? I've been trying to do that in my nginx conf, but so far i got no result. Here is my nginx config:

```
server {
                server_name raspberrypi.ae.local
                            192.168.1.*;

                location /flood/api {
                  proxy_pass http://127.0.0.1:3000;
                }

                location /flood/ {
                  alias /usr/lib/node_modules/flood/dist/assets/;
                  try_files $uri /flood/index.html;
                }

                rewrite (/flood)$ $1/login last;
        }
````

**Comment by jdarias** [comment]
I changed the rewrite rule as this, it is working so far:
```
rewrite ^/flood$ /flood/ permanent;
```

**Comment by jesec** [comment]
I didn’t try hard to do that when I wrote the tutorial. I think someone mentioned it in the Discord several weeks ago, and I forgot to update the wiki page. 

Anyways, good job figuring that out yourself and thanks for the wiki contributions. Additionally, I think you can probably use the redirect statement, as that’s `marginally` more efficient than rewrite.
